### DISCUSSION TITLE
Is there a way to stash hunks/lines?

### DISCUSSION THREAD

**Discussion by Sami6720** [jesseduffield/lazygit#3462] [Q&A] [answered]
I often commit by picking lines from a file in which I have made changes. Sometimes I would also like to stash certain lines/hunks on which I have made changes. Is that possible?

**Comment by mark2185** [comment, marked as answer]
Sure, press `S` in the `Files` panel to open up a menu.
![image](https://github.com/jesseduffield/lazygit/assets/8365241/f33810dc-6c02-4905-963d-6ba7d3e1d440)

**Reply by Sami6720** [reply]
Thanks!
